english/Raiders/LetsgoRaiders.md

```diff
@@ -1,3 +1,7 @@
 Lets go Raiders!
 
-[Here's the link](https://www.google.com)
\ No newline at end of file
+[Here's the link](https://www.google.com)
+
+You see a dark room
+
+[Walk into room](english/amnesia/amnesia.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.